test/hyperclient/link_test.rb

```diff
@@ -301,7 +301,7 @@ module Hyperclient
           end
 
           resource.foos._embedded.orders.first.id.must_equal 1
-          resource.foos.first.must_equal nil
+          resource.foos.first.must_be_nil
```

Comment:
`must_equal nil` would cause:

```
DEPRECATED: Use assert_nil if expecting nil from /Users/yuki/GitHub/hyperclient/test/hyperclient/link_test.rb:304. This will fail in Minitest 6.
```